### PR TITLE
Update HLT trigger listings

### LINKED ISSUE #2899: CMS: 2011 RunB datasets point to wrong HLT configs
For "Data taking / HLT", CMS 2011 RunB datasets (e.g. http://opendata.cern.ch/record/282) point to 2012 HLT config listing http://opendata.cern.ch/record/1701
Should be http://opendata.cern.ch/record/1700  (but that should be completed to include RunB, now it is only RunA).

### PR DESCRIPTION
**Pull request by katilp**
(closes #2899 and #1298 and #2942  )

- renames the json file for 2011 HLT config listing and adds the ones for RunB to the list
- corrects the link to the HLT listing for 2011 RunB datasets
- removes "Here is" from the description of HLT config listing pages

- adds two configs for the HI run period (with a wording indicating that the run range and CMSSW version are approximate)

To-do:
- In 2011 listing: needs trigger HLT configuration record numbers (FIXMEs), the records already exist.
- In 2010 listing: build records for the two new configs and fix their recids (now 0000)
  - /cdaq/physics/Run2010HI/v1.5/HIHLT/V1 : file from https://cernbox.cern.ch/index.php/s/Nut0NLvXYMaXEeb
  - /cdaq/physics/Run2010HI/v1.7/HIHLT/V1 : file from https://cernbox.cern.ch/index.php/s/C5zILNw5CxH6Koo
- I've asked to create the relevant CMSSW version tags for 2011 HI runs, when available check links

### CODE REVIEW

File: cernopendata/modules/fixtures/data/records/cms-trigger-information-2012.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
...3/v2.0/HLT/V3</a></td></tr>\n      <tr><td>199960 - 200381</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_5_2_6_onlpatch2\">CMSSW_5_2_6_onlpatch2</a></td><td><a href=\"/record/6142\">/cdaq/physics/Run2012/7e33/v4.0/HLT/V4</a></td></tr>\n      <tr><td>207214 - 207518</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_5_2_7_onlpatch3\">CMSSW_5_2_7_onlpatch3</a></td><td><a href=\"/record/6143\">/cdaq/physics/Run2012/8e33/v2.1/HLT/V5</a></td></tr>\n      <tr><td>193093 - 193124</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_5_2_4_onlpatch1\">CMSSW_5_2_4_onlpatch1</a></td><td><a href=\"/record/6144\">/cdaq/physics/Run2012/5e33/v4.16/HLT/V14</a></td></tr>\n      <tr><td>199318 - 199608</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_5_2_4_onlpatch5\">CMSSW_5_2_4_onlpatch5</a></td><td><a href=\"/record/6145\">/cdaq/physics/Run2012/7e33/v3.1/HLT/V2</a></td></tr>\n      <tr><td>200466 - 202504</td><td><a href=\"https://github.com/cms-sw/cmssw-cvs/tree/CMSSW_5_2_6_onlpatch3\">CMSSW_5_2_6_onlpatch3</a></td><td><a href=\"/record/6146\">/cdaq/physics/Run2012/7e33/v4.1/HLT/V1</a></td></tr>\n      </tbody>\n      </table>\n      "
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/cms-trigger-information-2012.json:4]: BTW for Run2011B datasets, in the provenance part, we still use terms such as "Configuration file for RECO step reco_2011A_BTag", pointing to records marked as "Configuration file for RECO step reco_2011A_BTag ". Some people may be confused that we seem advertise only 2011A everywhere.  

Example: http://opendata-dev.cern.ch/record/270 pointing to http://opendata-dev.cern.ch/record/3603

Is this OK? We could perhaps leave out "A" if it makes sense...

**Comment by tiborsimko** [comment]
> In 2011 listing: needs trigger HLT configuration record numbers (FIXMEs), the records already exist.

Added all links except one: `/cdaq/physics/Run2011/5e33/v1.4/HLT/V4` does not seem to exist as a record.
Note that we have V3, not V4: http://opendata.cern.ch/record/3571

The V4 file is well present on EOS though:

```console
$ eos ls -l /eos/opendata/cms/configuration-files/2011/cdaq_physics_Run2011_5e33_v1.4_HLT_V4.py
-rw-r-----   2 simko    us            1795880 Jul 21 14:24 cdaq_physics_Run2011_5e33_v1.4_HLT_V4.py
```

Is V4 the correct one to use? If so, I'll create a new record for this file.

File: cernopendata/modules/fixtures/data/records/cms-trigger-information-2012.json (hunk 1 above)
Comment (katilp) [reply cernopendata/modules/fixtures/data/records/cms-trigger-information-2012.json:4]: Yes, dropping A would make sense as the name is ours in any case. Thanks for spotting that.

**Comment by katilp** [comment]
Yes, please create a record. In https://fwyzard.web.cern.ch/fwyzard/hlt/2011/summary we have both:

```
CMSSW_4_2_9_HLT3_onlpatch2_ONLINE /cdaq/physics/Run2011/5e33/v1.4/HLT/V3                                           (runs 178420 - 178479)
CMSSW_4_2_9_HLT3_onlpatch2_ONLINE /cdaq/physics/Run2011/5e33/v1.4/HLT/V4                                           (runs 178667 - 178708)
```

**Comment by tiborsimko** [comment]
> In 2010 listing: build records for the two new configs and fix their recids (now 0000) 

Added two new configuration file records, updated links. Please check them here in `cms-configuration-files-HIRun2010.json`, e.g.  2.78TeV, PbPb, keyword heavy-ion physics...

**Comment by tiborsimko** [comment]
> Yes, please create a record.

All done. New records are 107, 108, 109.

Everything should be ready except for GitHub CMS SW links. Do we merge and deploy on DEV and address the GitHub CMS SW links later?

File: cernopendata/modules/fixtures/data/records/cms-trigger-information-2012.json (hunk 1 above)
Comment (tiborsimko) [reply cernopendata/modules/fixtures/data/records/cms-trigger-information-2012.json:4]: OK, dropped A everywhere, from Run2011A and Run2011B records.

**Comment by katilp** [comment]
Yes, the code still needs further testing and documentation, so we can address the final links later. Thanks!

**Review by tiborsimko — APPROVED** [review]
Merging with the changes, to be tested on DEV, GitHub links to be completed later.

**Pull request merged**